media/css/products/vpn/resource-center.scss

```diff
@@ -0,0 +1,193 @@
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: 90% 100%;
+    }
+    @media #{$mq-lg} {
+        background-position: 45% $layout-xs;
+        background-size: auto;
+    }
+}
+
+.resource-center-hero {
+    background-color: $color-white;
+}
+
+.resource-center-wordmark {
+    @include image-replaced();
+    @include background-size(94px 32px);
+    background: transparent url('/media/img/logos/vpn/logo-with-wordmark.svg') top left no-repeat;
+    height: 32px;
+    margin: 0 auto $spacing-lg;
+    width: 94px;
+
+    @media #{$mq-md} {
+        margin: 0 0 $spacing-xl;
+    }
+
+}
+
+.resource-center-cta {
+    margin-bottom: $spacing-md;
+}
+
+.resource-center-split-media .mzp-c-split-media-asset {
+    // smaller image on small breakpoints
+    max-width: 80%;
+
+    @media #{$mq-md} {
+        max-width: 100%;
+    }
+}
+
+.resource-center-cta-title {
+    margin-bottom: $spacing-2xl;
+}
+
+.resource-center-cta-desc {
+    @include text-body-sm;
+    @include font-base;
+    color: $color-marketing-gray-60;
+}
+
+// temporary until we incorporate breadcrumb to protocol
+.mzp-c-breadcrumb {
+    background-color: $color-light-gray-20;
+    margin-bottom: 0;
+    padding: $spacing-sm $spacing-md;
+
+    .mzp-c-breadcrumb-item, .mzp-c-breadcrumb-title {
+        display: inline-block;
+        font-weight: 400;
+
+        .mzp-c-breadcrumb-item-link {
+            text-decoration: none;
+        }
+    }
+
+    .mzp-c-breadcrumb-item {
+        font-weight: bold;
+        &:before {
+            @include bidi(((content, "\2192", "\2190"),));
+            font-weight: normal;
+            margin: 0 0.25em;
+        }
+    }
+
+    @media #{$mq-md} {
+        padding: $spacing-sm $layout-xl - $spacing-md;
+    }
+
+}
+
+.vpn-article-container {
+    background: url('/media/img/products/vpn/resource-center/resource_center_article_gradient.svg');
+    background-repeat: no-repeat;
+    background-size: auto;
+    // The background image will cut off on larger viewports
+    @media #{$mq-lg} {
+        background-size: contain;
+    }
+}
+
+.vpn-article {
+    background-color: $color-white;
+    box-sizing: border-box;
+    padding: $layout-sm $layout-md 0 $layout-md;
+
+    .vpn-article-category {
+        @include text-body-xs;
+        background: $color-black;
+        border-radius: $border-radius-sm;
+        color: $color-white;
+        display: inline-block;
+        font-weight: 300;
+        padding: $spacing-xs $spacing-md $spacing-xs $spacing-md;
+        text-transform: uppercase;
+    }
+
+    .vpn-article-date {
+        align-items: center;
+        display: flex;
+
+        .date-icon {
+            @include bidi(((margin-right, $spacing-sm, margin-left, $spacing-sm),));
+        }
+    }
+
+    h3 {
+        @include text-title-sm;
+    }
+}
+
+.vpn-article-bottom-title {
+    margin-bottom:$spacing-xl;
+}
+
+.mzp-c-hr {
```

Comment:
this should probably have a vpn-specific classname, as we're overriding [protocol base hr styles](https://github.com/mozilla/protocol/blob/6b23d595c8a9a98311a729b6039ef11ae7041a9c/src/assets/sass/protocol/base/elements/_common.scss#L25)

Reply:
Oh, good call!